### ISSUE TITLE
Prototype unified AV player with placeholder

### ISSUE THREAD

**Issue by jamesdonoh** [bbc/simorgh#2422]
**Is your feature request related to a problem? Please describe.**
Initialising the player (SMP) for audio and video content has a large and immediate impact on page weight, e.g. `c0000000030o` has a total content size of 2.7MB (uncompressed) compared with 890KB when the player is disabled. This happens regardless of whether the user interacts with the player. We want to find a way of minimising the cost/performance impact on users without adversely affecting the user journeys for playing AV content. We think we can do this by delaying the embedding of the video till the user interacts with it.

We are also now moving to a unified approach for embedding AV content which uses the same approach for both AMP and canonical.

We need to prototype the front-end parts of this solution to ensure that it meets our UX, performance and a11y requirements.

**Describe the solution you'd like**
- Implement AV player component that supports both AMP and canonical
  - Use dummy embed URL for testing purposes as was done in https://github.com/bbc/simorgh/pull/1774
- Placeholder:
  - Canonical: implement a simple placeholder component for each piece of AV content using a static dummy image that 'upgrades' into a full player when the user clicks on it (see @rhell 's example at https://github.com/bbc/simorgh/pull/2550)
  - AMP: use the built-in placeholder behaviour provided by AMP custom elements
- (Canonical) evaluate ways of eliminating the need for the user to click multiple times (e.g. by using autoplay)
- The solution needs to be cross-browser compatible
- The solution needs to be fully accessible
    - Refer to screen reader UX in https://app.zeplin.io/project/5baa564c05c2eba0d2b89568/screen/5c596fa88971d341fae3fd53
- Summarise progress and main challenges found on the PR for review along with an evaluation of any options available

NB out of scope:
- UX finesse/formatting
- Unit tests
- Real data

**Describe alternatives you've considered**
We could stick with the existing solution or wait for such functionality to be available in the player itself.

**Testing notes**
- Must be a11y swarmed
- Must be cross-browser compatible

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

- This should not be merged until A11Y swarm is done and tests have been carried out.

**Comment by jamesbhobbs** [comment]
Interesting reading related to this, also showing it's possible in chrome https://developers.google.com/web/updates/2017/09/autoplay-policy-changes

**Comment by 12** [comment]
Here's a very basic (but working) implementation we could pursue: https://codepen.io/rhellyar/pen/PrrPgW

**Comment by jamesdonoh** [comment]
Blocked on https://github.com/bbc/simorgh/issues/2614
